resolwe_bio/processes/support_processors/library_strandedness.yml

```diff
@@ -78,15 +78,16 @@
         {% set reads_input = '-1 mate1.fastq -2 mate2.fastq' %}
       {% endif %}
 
-      salmon quant \
+      salmon --no-version-check quant \
```

Comment:
Try to re-use the script `detect_strandedness.sh` in this process.

Reply:
This process operates on raw fastq data, the `detect_strandedness.sh` script is written with post-alignment needs in mind